src/Entity/Alias.php

```diff
@@ -13,7 +13,7 @@
 /**
  * @author louis <louis@systemli.org>
  */
-class Alias implements SoftDeletableInterface
```

Comment:
Why? This reverted the complete soft delete functionality.

Reply:
No, it doesn't. The actual functionality is in `DeleteTrait`.

Reply:
But we check in BaseRepository if the Class is instance of the Interface.

Reply:
I see. I did this because of #61. Now it works similar to the User Entity.

Reply:
lets do it in another PR